lib/ActivityStreams.js

```diff
@@ -411,6 +432,17 @@ ActivityStreams.prototype = {
       this._prefsProvider.actionHandler(args);
     };
     this.on(CONTENT_TO_ADDON, this._contentToAddonHandlers);
+
+    simplePrefs.on("", prefName => {
```

Comment:
Right now there isn't really a convenient way to use `PrefsProvider` for this use case – it's more for handling communication with content, although we could add something to do it (i.e. an `addListener`/`on` method) – the advantage of that would be we could handle removing listeners on unload in once place.

For now, this seems reasonable (we could file the enhancement as separate ticket) -- although this method needs to be cached on the instance and removed at unload time.